sonar-flex-plugin/pom.xml

```
@@ -11,7 +11,7 @@
   <artifactId>sonar-flex-plugin</artifactId>
   <packaging>sonar-plugin</packaging>
 
-  <name>SonarFlex :: SonarQube Plugin</name>
+  <name>SonarSource Flex analyzer :: SonarQube Plugin</name>
   <description>Code Analyzer for Flex / ActionScript</description>
```

@amelie-renard-sonarsource what do you think about?
```suggestion
  <description>Code Quality and Security for Flex / ActionScript</description>
```

---

I kept this wording to be consistent with other languages (eg php, cobol) but yes, your suggestion makes more sense